dev/dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM sequenceiq/spark:1.6.0
```

Comment:
sequenceiq/spark didn't release version 1.5.2. 
Version 1.5.1 is unavailable now.
So I can only use version 1.6.0.